jobs/build_docker/prepare_docker_post_test.sh

```diff
@@ -61,3 +61,18 @@ set -e
 
 docker-compose -f docker-compose-mini.yml up > $WORKSPACE/build-log/vagrant.log &
 popd
+
+mountpath="$WORKSPACE/RackHD/docker/files/mount"
+retrytimes=5
+while [ ! -d "$mountpath" ]
+do
+    echo "mount is not finished"
+    retrytimes=$(($retrytimes-1))
+    echo "retry : $retrytimes times"
+    sleep 10
+    if [ $retrytimes -eq 0 ]; then
+        break
+    fi
```

Comment:
I'm sorry.
if the retrytimes equal to 0 and the mount path still not exist.
It's still useless to chown in line 78..